resources/definitions/fdmprinter.def.json

```diff
@@ -2569,6 +2599,20 @@
                             "limit_to_extruder": "top_bottom_extruder_nr",
                             "settable_per_mesh": true
                         },
+                        "jerk_sanding":
+                        {
+                            "label": "Sanding Jerk",
+                            "description": "The maximum instantaneous velocity change while performing sanding.",
```

Comment:
Yeah, I think I forgot them because they aren't in the same category as the rest of the ironing settings.

I fixed the setting names and descriptions here: https://github.com/Ultimaker/Cura/pull/1978/commits/a7ed0d011ad60f878629b86a1df6113718c0423a
I fixed the setting IDs here: https://github.com/Ultimaker/Cura/pull/1978/commits/092483cac5e7d6fb478502c3fca666d60627b905 and in the accompanying commit in CuraEngine: https://github.com/Ultimaker/CuraEngine/pull/534/commits/d927f7317a2601eb90f650e936377d3f7aa5b360